### PR TITLE
docs update

### PR DESCRIPTION
**Pull request by sidnt**
elucidate `run` and `fold`

### CODE REVIEW

**Comment by ghostdogpr** [comment]
Useful addition, thanks! How about moving it to the text below or above the code? I think it would be more readable than as a code comment?

File: docs/getting_started.md [hunk 1]
```diff
@@ -44,6 +44,18 @@ object MyApp extends App {
     } yield ()
 }
 ```
+`run` should return a ZIO value which has all its errors handled,  
+which, in zio parlance, is aka an unexceptional ZIO value.  
+
+One way to do that, is to invoke `fold` over a ZIO value, to get an unexceptional ZIO value  
+That requires two handler functions, one `eh: E => B`, other `ah: A => B`  
+If myAppLogic fails, `eh` will be used to get from e:E to b:B  
+if it suceeds, `ah` will be used to get from a:A to b:B
```
Comment (ghostdogpr) [inline docs/getting_started.md]: typo: `succeeds`

Comment (ghostdogpr) [inline docs/getting_started.md]: Maybe for a nicer flow start all sentences with an uppercase letter and end with a dot e.g. "If it succeeds, `ah` will be used to get from a:A to b:B."

Comment (sidnt) [reply docs/getting_started.md]: This is a bit custom philosophy. 'Fullstop capital letter' sentence breaking pattern is used if it's deemed to be more meaningful. But here, I didn't intend to break the sentence, and yet give a pause, without making the sentence look too long. Hence ..

File: docs/getting_started.md [hunk 2]
```diff
@@ -44,6 +44,18 @@ object MyApp extends App {
     } yield ()
 }
 ```
+`run` should return a ZIO value which has all its errors handled,  
+which, in zio parlance, is aka an unexceptional ZIO value.
```
Comment (iravid) [inline docs/getting_started.md]:
```suggestion
which, in ZIO parlance, is an unexceptional ZIO value.  
```

File: docs/getting_started.md [hunk 3]
```diff
@@ -44,6 +44,18 @@ object MyApp extends App {
     } yield ()
 }
 ```
+`run` should return a ZIO value which has all its errors handled,  
+which, in zio parlance, is aka an unexceptional ZIO value.  
+
+One way to do that, is to invoke `fold` over a ZIO value, to get an unexceptional ZIO value
```
Comment (iravid) [inline docs/getting_started.md]:
```suggestion
One way to do that, is to invoke `fold` over a ZIO value, to get an unexceptional ZIO value.
```

File: docs/getting_started.md [hunk 4]
```diff
@@ -44,6 +44,18 @@ object MyApp extends App {
     } yield ()
 }
 ```
+`run` should return a ZIO value which has all its errors handled,  
+which, in zio parlance, is aka an unexceptional ZIO value.  
+
+One way to do that, is to invoke `fold` over a ZIO value, to get an unexceptional ZIO value  
+That requires two handler functions, one `eh: E => B`, other `ah: A => B`
```
Comment (iravid) [inline docs/getting_started.md]:
```suggestion
That requires two handler functions: `eh: E => B` and `ah: A => B`.
```

File: docs/getting_started.md [hunk 5]
```diff
@@ -44,6 +44,18 @@ object MyApp extends App {
     } yield ()
 }
 ```
+`run` should return a ZIO value which has all its errors handled,  
+which, in zio parlance, is aka an unexceptional ZIO value.  
+
+One way to do that, is to invoke `fold` over a ZIO value, to get an unexceptional ZIO value  
+That requires two handler functions, one `eh: E => B`, other `ah: A => B`  
+If myAppLogic fails, `eh` will be used to get from e:E to b:B
```
Comment (iravid) [inline docs/getting_started.md]:
```suggestion
If myAppLogic fails, `eh` will be used to get from `e: E` to `b: B`;
```

File: docs/getting_started.md (hunk 1 above)
Comment (iravid) [reply docs/getting_started.md]:
```suggestion
if it succeeds, `ah` will be used to get from `a: A` to `b: B`. 
```

File: docs/getting_started.md [hunk 6]
```diff
@@ -44,6 +44,18 @@ object MyApp extends App {
...
 ```
+`run` should return a ZIO value which has all its errors handled,  
+which, in zio parlance, is aka an unexceptional ZIO value.  
+
+One way to do that, is to invoke `fold` over a ZIO value, to get an unexceptional ZIO value  
+That requires two handler functions, one `eh: E => B`, other `ah: A => B`  
+If myAppLogic fails, `eh` will be used to get from e:E to b:B  
+if it suceeds, `ah` will be used to get from a:A to b:B  
+
+myAppLogic, as folded above, produces a unexceptional ZIO value, with B being Int
```
Comment (iravid) [inline docs/getting_started.md]:
```suggestion
`myAppLogic`, as folded above, produces an unexceptional ZIO value, with `B` being `Int`.  
```

File: docs/getting_started.md [hunk 7]
```diff
@@ -44,6 +44,18 @@ object MyApp extends App {
...
+`run` should return a ZIO value which has all its errors handled,  
+which, in zio parlance, is aka an unexceptional ZIO value.  
+
+One way to do that, is to invoke `fold` over a ZIO value, to get an unexceptional ZIO value  
+That requires two handler functions, one `eh: E => B`, other `ah: A => B`  
+If myAppLogic fails, `eh` will be used to get from e:E to b:B  
+if it suceeds, `ah` will be used to get from a:A to b:B  
+
+myAppLogic, as folded above, produces a unexceptional ZIO value, with B being Int  
+If myAppLogic fails, (for whatever reasons), there will be a 1, it it succeeds, there will be a 0
```
Comment (iravid) [inline docs/getting_started.md]:
```suggestion
If `myAppLogic` fails, there will be a 1; if it succeeds, there will be a 0.
```

**Comment by sidnt** [comment]
why isn't this update being reflected on the zio microsite?

**Comment by ghostdogpr** [comment]
@sidnt we deploy the microsite when we release a new version, so that the docs (which often refer to some code) reflect the state of the last version.

**Pull request merged**